apps/indexer/src/services/url-resolver.ts

```diff
@@ -1,5 +1,8 @@
 import { LRUCache } from "lru-cache";
-import { createURLResolver, type ResolvedURL } from "../resolvers/url-resolver";
+import {
+  createURLResolver,
+  type ResolvedURL,
+} from "../resolvers/url-resolver.ts";
```

Comment:
why do we suddenly need to added the extension?

Reply:
node --experimental-strip-types support.